### ISSUE TITLE
UI: Home Layout Setup

### ISSUE THREAD

**Issue by forrestfiller** [ryanab/fileshare#7]
- If user isn't logged in, **don't** render files

- Once user is logged in:
    - render all files on cards (one file per card) with newest on top, 10 at a time 'click to load more'       button at the bottom
    - in thin sidebar on left:
        - render a small "plus" dropzone box
        - render username
        - render last three uploaded files with timestamp
    - If user clicks on the navbar filters (pdf, image, video, misc),  render only that type of file

**Comment by jdrichardstech** [comment]
dropzone zone needs to be a button. non image files don't seem to upload by dragging into the box. user needs to choose the file after clicking on dropzone.

**Comment by forrestfiller** [comment]
@jdrichardstech so is that a bug? 
 - Or is it fine?
  - **Would you rather have no drag and drop box, with traditional file selection only?**

**Comment by jdrichardstech** [comment]
I don't know if it is a bug. However since we are uploading many file types my personal choice would be to avoid the drag & drop and opt for traditional file selection. See what others think.

**Comment by forrestfiller** [comment]
@jdrichardstech agreed. Ill eliminate the drag and drop for now, and we can always bring it back if we have any issues. Thanks for your help!
